src/components/profile/Note.js

```diff
@@ -1,9 +1,74 @@
 import './profile.css';
+import { Button } from '@mui/material';
+import { useContext, useState } from 'react';
+import client from '../../utils/client';
+import { loggedInUserContext } from '../../Helper/loggedInUserContext';
+import { useParams } from 'react-router-dom';
+import { formatDate } from '../../Helper/dateFormatter';
+
+export default function Note({ note, setNotes }) {
+  const params = useParams();
+  const [isDeleting, setIsDeleting] = useState(false)
+  const { loggedInUser } = useContext(loggedInUserContext);
+  const [noteEdit, setNoteEdit] = useState({ content: '' });
+  const [checkIfEditing, setCheckIfEditing] = useState(false)
+
+  const deleteNoteHandler = (event, noteId) => {
+    event.preventDefault();
+    if(isDeleting){
+      client
+        .delete(`/note/${noteId}`)
+        .then(() => setIsDeleting(false))
+        .catch((err) => console.error(err.response))
```

Comment:
is there a reason why the catch is not at the end of the `then` chain?

Reply:
There is no reason – didn't know ``.catch``should be in the end

Reply:
moved the catch to the end of the chain